### PR TITLE
Address design/copy comments for NP 

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/0/1205258659015741/f

### Description
See [asana task](https://app.asana.com/0/0/1205258659015741/f)

### Steps to test this PR
QA optional (see [asana task](https://app.asana.com/0/0/1205258659015741/f))

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3457** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3457" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3457?utm_source=stack-comment).

File: app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/DeviceInfoCollector.kt [hunk 1]
```diff
@@ -62,9 +61,7 @@ object DeviceInfoCollectorModule {
         return runCatching {
             context.packageName?.let {
                 val powerManager = context.getSystemService(Context.POWER_SERVICE) as PowerManager
-                powerManager.isIgnoringBatteryOptimizations(context.packageName).also {
-                    logcat { "aitor $it" }
-                }
```
Comment (aitorvs) [inline app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/DeviceInfoCollector.kt:67]: This was a left over from previous PR

File: app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt [hunk 2]
```diff
@@ -291,20 +291,26 @@ class SettingsActivity : DuckDuckGoActivity() {
...
+                            CheckListItem.CheckItemStatus.ENABLED
+                        } else {
+                            CheckListItem.CheckItemStatus.WARNING
+                        }
+                        netpPSetting.setItemStatus(netPItemStatus)
                     } else {
-                        CheckListItem.CheckItemStatus.WARNING
+                        netpPSetting.show()
+                        netpPSetting.setSecondaryText(getString(R.string.netpSettingsNeverEnabled))
+                        netpPSetting.setItemStatus(CheckListItem.CheckItemStatus.DISABLED)
```
Comment (aitorvs) [inline app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt:312]: This was feedback from [here](https://app.asana.com/0/0/1205230811122378/1205255564162048/f)

File: app/src/main/res/layout/view_item_downloads_notify_me.xml [hunk 3]
```diff
@@ -19,6 +19,7 @@
     android:id="@+id/downloadsNotifyMe"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:layout_margin="@dimen/keyline_4"
```
Comment (aitorvs) [inline app/src/main/res/layout/view_item_downloads_notify_me.xml:22]: Removing the intrinsic margin from the notify me component as it makes it more difficult to have proper margins when designing the screen.

This keyline has been moved to the screens that embed the component

File: network-protection/network-protection-api/src/main/java/com/duckduckgo/networkprotection/api/NetworkProtectionWaitlist.kt [hunk 4]
```diff
@@ -37,6 +37,6 @@ interface NetworkProtectionWaitlist {
         object NotUnlocked : NetPWaitlistState()
         object JoinedWaitlist : NetPWaitlistState()
         object PendingInviteCode : NetPWaitlistState()
-        object InBeta : NetPWaitlistState()
+        data class InBeta(val termsAccepted: Boolean) : NetPWaitlistState()
```
Comment (aitorvs) [inline network-protection/network-protection-api/src/main/java/com/duckduckgo/networkprotection/api/NetworkProtectionWaitlist.kt:40]: Addresses [this comment](https://app.asana.com/0/0/1205230811122378/1205255564162048/f)

File: network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt [hunk 5]
```diff
@@ -85,4 +85,6 @@ enum class NetworkProtectionPixelNames(
     NETP_WAITLIST_NOTIFICATION_CANCELLED("m_netp_ev_waitlist_notification_cancelled_c"),
     NETP_WAITLIST_NOTIFICATION_LAUNCHED_DAILY("m_netp_ev_waitlist_notification_launched_d"),
     NETP_WAITLIST_NOTIFICATION_LAUNCHED("m_netp_ev_waitlist_notification_launched_c"),
+    NETP_WAITLIST_BETA_ENABLED_DAILY("m_netp_ev_waitlist_enabled_d"),
+    NETP_WAITLIST_BETA_ENABLED("m_netp_ev_waitlist_enabled_c"),
```
Comment (aitorvs) [inline network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt:89]: This is the same as `m_netp_imp_settings_entry_[c/d]` from [here](https://app.asana.com/0/0/1205145153997698/f). But with different (better) naming for Android

Comment (karlenDimla) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt:89]: I assume we would want to add this in `NetpCohortPixelInterceptor`.

Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt:89]: Ah, great catch! Will do

**Review by karlenDimla — APPROVED** [review]
Aside from https://github.com/duckduckgo/Android/pull/3457#discussion_r1293289543, everything looks good.

**Pull request merged**